src/backend/x11/fallback.rs

```diff
@@ -0,0 +1,100 @@
+//! The fallback screenshot creator if all previous attempts failed to create the screenshots of a monitor.
```

Comment:
Desktop Portals are independent of Display protocol. We will need them for gnome on waylant too.

Reply:
Oh fair, I'll dig into it